languages/java/src/main/java/com/bitwarden/sdk/BitwardenClient.java

```diff
@@ -39,36 +41,20 @@ public BitwardenClient(BitwardenSettings bitwardenSettings) {
         commandRunner = new CommandRunner(library, client);
         projects = new ProjectsClient(commandRunner);
         secrets = new SecretsClient(commandRunner);
+        auth = new AuthClient(commandRunner);
         isClientOpen = true;
     }
 
     static <T, R> Function<T, R> throwingFunctionWrapper(ThrowingFunction<T, R, Exception> throwingFunction) {
-
         return i -> {
             try {
                 return throwingFunction.accept(i);
             } catch (Exception ex) {
-                throw new BitwardenClientException("Response deserialization failed");
+                throw new BitwardenClientException("Response deserialization failed", ex);
```

Comment:
Actually since it's multi purpose now, let's change the message
```suggestion
                throw new BitwardenClientException("Response failed", ex);
```

Reply:
Fair, updated in https://github.com/bitwarden/sdk/pull/1005/commits/16484df6301a14ca79dc7375907576051d3d720a